docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Paddle.md

```diff
@@ -22,6 +22,10 @@ A summary of the steps for optimizing and deploying a model trained with Paddle\
 |OCRNET| semantic segmentation| Models are exported from [PaddleSeg](https://github.com/PaddlePaddle/PaddleSeg/tree/release/2.1). Refer to [model_export.md](https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.1/docs/model_export.md#)|
 |Yolo v3| detection| Models are exported from [PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.1). Refer to [EXPORT_MODEL.md](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/deploy/EXPORT_MODEL.md#).|
 |ppyolo| detection| Models are exported from [PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.1). Refer to [EXPORT_MODEL.md](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.1/deploy/EXPORT_MODEL.md#).|
+|MobileNet-SSDLite| detection| Models are exported from [PaddleDetection](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.2). Refer to [EXPORT_MODEL.md](https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.2/deploy/EXPORT_MODEL.md#).|
```

Comment:
Better narrow down to "MobileNetv3-SSD" instead of SSDLite. MobilenetV3 with FPN is not enabled. 
https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.2/static/docs/MODEL_ZOO_cn.md#ssdlite

Reply:
updated